### DISCUSSION TITLE
1.22 deployment

### DISCUSSION THREAD

**Discussion by mstrohl** [argoproj/argo-rollouts#1556] [Q&A]
Hi,

I wanted to get argo rollouts on my 1.22 cluster but it seem that the controller is not able to watch ingresses. so I'm trying to upgrade librairies to build a compliant image but it fails on go build. any idea or guidelines to make it work on my cluster?
FYI :
K8S version : 1.22
ingress controller : ingress

Let me know if you need more

**Comment by kostis-codefresh** [comment]
Hello.

Have you seen these?

* https://github.com/argoproj/argo-rollouts/issues/1507
* https://github.com/argoproj/argo-rollouts/pull/1529
